docs/userguides/departingemployee.md

```diff
@@ -1,49 +1,28 @@
-# Search and Add user to Departing Employee List
+# Add or Remove user to Departing Employee List
 
-Let's say you want to add a user to the departing employee list and all you know is their user UID. We can achieve
-this using the `py42.sdk.users.get_by_uid()` method to recover their username, and then use the
-`py42.sdk.detectionlists.departing_employee.create()` method which requires a username as a parameter.
+Let's say you want to add a user to the departing employee list then all you need to know is their user UID.
 
-This is how you can get the username from the user UID:
-
-```python
-user = sdk.users.get_by_uid("895005723650937319")
-username = user["username"]
-```
-
-Next, use the `py42.sdk.detectionlists.departing_employee.create()` method to add that user to the departing employee
-list. Take note of its case ID if you want to do anything else right away:
+`user_id` below refers to user UID.
```

Comment:
Can you show how to get the userUid using `sdk.user.get_by_username()`

Reply:
Added.